scripts/windows/image/installers/install-sccache.ps1

```diff
@@ -0,0 +1,10 @@
+
+# First install scoop
+Set-ExecutionPolicy RemoteSigned -Scope CurrentUser
+iex "& {$(irm get.scoop.sh)} -RunAsAdmin"
```

Comment:
`.sh` wat

Reply:
This is getting [scoop](https://scoop.sh/) which is the one easy way of installing sccache on windows.

We are in an elevated shell so we need to tell it to run as admin

Reply:
The line is from https://github.com/ScoopInstaller/Install#for-admin